### PR TITLE
Adding fields 'aws_log_group' and 'aws_log_stream' to flow log and raw log codecs

### PR DESCRIPTION
**Pull request by danielgrant**
This pull request addresses #54.

### CODE REVIEW

File: src/main/java/org/graylog/aws/inputs/codecs/CloudWatchFlowLogCodec.java [hunk 1]
```diff
@@ -51,6 +51,8 @@ public Message decodeLogData(@Nonnull final CloudWatchLogEvent logEvent) {
                     flowLogMessage.getTimestamp()
             );
             result.addFields(buildFields(flowLogMessage));
+            result.addField("aws_log_group", logGroup);
```
Comment (joschi) [inline src/main/java/org/graylog/aws/inputs/codecs/CloudWatchFlowLogCodec.java]: Please extract the key names as constants into the existing `AWS` class and use them in `CloudWatchFlowLogCodec` and in `CloudWatchRawLogCodec`.

File: src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java [hunk 2]
```diff
@@ -31,6 +31,13 @@
  * </pre>
  */
 public class CloudWatchLogData {
+
     @JsonProperty("logEvents")
     public List<CloudWatchLogEvent> logEvents;
+
+    @JsonProperty("logGroup")
+    public String logGroup;
```
Comment (joschi) [inline src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java]: Can this ever by `null` (i. e. not exist)?

File: src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java [hunk 3]
```diff
@@ -31,6 +31,13 @@
...
+
+    @JsonProperty("logStream")
+    public String logStream;
```
Comment (joschi) [inline src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java]: Can this ever by `null` (i. e. not exist)?

File: src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java (hunk 2 above)
Comment (danielgrant) [reply src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java]: Looking at the [documentation and example](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/SubscriptionFilters.html), there is nothing to suggest that this field is optional / might not be present in any situation. Additionally, in all of my testing to date it has never not been present.

File: src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java (hunk 3 above)
Comment (danielgrant) [reply src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java]: Looking at the [documentation and example](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/SubscriptionFilters.html), there is nothing to suggest that this field is optional / might not be present in any situation. Additionally, in all of my testing to date it has never not been present.

File: src/main/java/org/graylog/aws/inputs/codecs/CloudWatchFlowLogCodec.java (hunk 1 above)
Comment (danielgrant) [reply src/main/java/org/graylog/aws/inputs/codecs/CloudWatchFlowLogCodec.java]: I've extracted these fields to constants.

**Comment by danielgrant** [comment]
Hi @joschi,

Thanks for the feedback.

With regards to the unit testing, I couldn't see any existing test classes / unit testing around the classes that I've modified when I was making these changes. I can build these from scratch, but other priorities elsewhere means that doing so will take a bit longer.

Cheers,
Daniel.

File: src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java (hunk 2 above)
Comment (joschi) [reply src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java]: ✅

File: src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java (hunk 3 above)
Comment (joschi) [reply src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java]: ✅

**Pull request merged**